imageroot/systemd/user/postgres.service

```diff
@@ -24,6 +24,9 @@ ExecStart=/usr/bin/podman run \
     --cidfile=%t/postgres.ctr-id \
     --cgroups=no-conmon \
     --volume=pgdata:/var/lib/postgresql/data \
+    --env POSTGRES_DB="webtop5" \
+    --env POSTGRES_USER="postgres" \
```

Comment:
I am not so sure if we need to explicitly declare `POSTGRES_USER` and `POSTGRES_PASSWORD`. Why were they not declared in the previous version?

Reply:
exactly, POSTGRES_USER=postgres is an option but for clarity I want to express it, Password now is mandatory